.github/workflows/docker.yml

```diff
@@ -19,32 +19,34 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-
+      - name: Modify Branch Name
+        run: echo "MODIFIED_REF_NAME=$(echo '${{ github.ref_name }}' | sed 's/[^a-zA-Z0-9]/-/g')" >> $GITHUB_ENV

```

Comment:
Maybe include some information about what we are doing here and why?

Reply:
added comments